test/e2e/chrome/metamask.spec.js

```diff
@@ -107,7 +102,7 @@ describe('Metamask popup page', function () {
 
     it('shows value was created and seed phrase', async () => {
       await delay(300)
-      seedPhase = await driver.findElement(By.css('.twelve-word-phrase')).getText()
+      await driver.findElement(By.css('.twelve-word-phrase')).getText()
```

Comment:
dont need to do this if we're not using the value

Reply:
maybe get the value and verify the content is 12 words
`seedPhrase.split(' ').length === 12`